eks_fargate/README.md

```diff
@@ -124,7 +124,178 @@ The setup below configures the Cluster Agent to communicate with the Agent sidec
 
    ```shell
    kubectl create secret generic datadog-secret -n datadog-agent \
-           --from-literal api-key=<YOUR_DATADOG_API_KEY> --from-literal token=<CLUSTER_AGENT_ TOKEN>
+           --from-literal api-key=<YOUR_DATADOG_API_KEY> --from-literal token=<CLUSTER_AGENT_TOKEN>
+   kubectl create secret generic datadog-secret -n fargate \
+           --from-literal api-key=<YOUR_DATADOG_API_KEY> --from-literal token=<CLUSTER_AGENT_TOKEN>
+   ```
+   For more information how these secrets are used, see the [Cluster Agent Setup][35].
+
+###### Setup
+
+1. Create a `DatadogAgent` custom resource in the `datadog-agent.yaml` with Admission Controller enabled:
+
+   ```yaml
+    apiVersion: datadoghq.com/v2alpha1
+    kind: DatadogAgent
+    metadata:
+      name: datadog
+    spec:
+      global:
+        clusterAgentTokenSecret:
+          secretName: datadog-secret
+          keyName: token
+        credentials:
+          apiSecret:
+            secretName: datadog-secret
+            keyName: api-key
+      features:
+        admissionController:
+          agentSidecarInjection:
+            enabled: true
+            provider: fargate
+   ```
+  Then apply the new configuration:
+  
+  ```shell
+  kubectl apply -n datadog-agent -f datadog-agent.yaml
+  ```
+
+2. After the Cluster Agent reaches a running state and registers Admission Controller mutating webhooks, an Agent sidecar is automatically injected into any pod created with the label `agent.datadoghq.com/sidecar:fargate`. 
+   **The Admission Controller does not mutate pods that are already created**.
+
+**Example result**
```

Comment:
Can you indent everything from lines 166-200 by 3 spaces? I think it'll line things up nicely.

Reply:
I've adjusted the indentation. Could you please review? :)

Reply:
https://github.com/DataDog/integrations-core/pull/17846/commits/601742ac609f9045da707e3ed1d2dacf65a0a647